src/zh/wnacg/src/eu/kanade/tachiyomi/extension/zh/wnacg/Preferences.kt

```diff
@@ -71,7 +71,7 @@ class UpdateUrlInterceptor(private val preferences: SharedPreferences) : Interce
 
         val failedResponse = try {
             val response = chain.proceed(request)
-            if (response.isSuccessful) return response
+            if (response.isSuccessful && response.peekBody(Long.MAX_VALUE).string().contains("紳士漫畫")) return response
```

Comment:
Could there be a cheaper check? What's the URL that returned successful code?

Reply:
At least check the content-type like this 
` if (response.header("content-type")?.startsWith("text/html") != true) {
            return response
        }`

Reply:
Please provide the problematic URL so we can decide what's the proper thing to do.

Reply:
Hello, sorry for the late reply. 

> At least check the content-type like this ` if (response.header("content-type")?.startsWith("text/html") != true) { return response }`

Hello, thanks for the suggestion. I did consider using that but I didn't think it would work, as the previous broken website such as `hm01.lol` does return `Content-Type: text/html`. But I might be wrong.

> Please provide the problematic URL so we can decide what's the proper thing to do.|

> Could there be a cheaper check? What's the URL that returned successful code?

Previous sites such as `hm01.lol`, `hm02.lol`, etc., are mostly parked, not dead, and they do return a status code of 200. And as for `cheaper check`, I'm not really sure, as this is what all I can think of to fix the issue, sorry.

> Doesn't this effectively blocks away all images? Did you test your changes?

Sorry, I'm not quite sure what you mean. What images are you referring to? All images loaded just fine on my phone. 

_I might've misunderstood your suggestion, and I'm sorry if I did, as English is not my primary language. Thank you. :pray:_

Reply:
```suggestion
            if (response.isSuccessful && response.header("Server") != "Parking/1.0") return response
```

Reply:
> At least check the content-type like this ` if (response.header("content-type")?.startsWith("text/html") != true) { return response }`

Hello, I’ve tried the code you suggested, and it seems the sites do return a `text/html` content type. 
I also used `reqbin` to confirm that this behaviour isn’t specific to my device.